transport/src/main/java/io/netty/bootstrap/ServerBootstrap.java

```diff
@@ -183,8 +287,8 @@ public void initChannel(Channel ch) throws Exception {
                 if (handler != null) {
                     pipeline.addLast(handler);
```

Comment:
I could,but it's a little tedious.

1. I need remove the default `ServerBootstrapAcceptor` netty added and replace it with an in house one and then set `setAutoRead(true)`.Or
2. I override the `channelRead(ChannelHandlerContext ctx, Object msg)` method in the handler which do the handling and do not forward the event to the next `ServerBootstrapAcceptor`.

For most case I think user may not figure out that the `channelRead` for the ServerChannel's handler will reaceive the accepted channel.I think this should help a little.